src/main/java/it/gov/pagopa/payhub/activities/mapper/exportflow/classifications/ClassificationsExportFlowFileDTOMapper.java

```diff
@@ -104,8 +107,8 @@ public ClassificationsExportFlowFileDTO map(ClassificationViewDTO retrievedObjec
                 .tresIuf(retrievedObject.getTresIuf())
                 .tresIuv(retrievedObject.getTresIuv())
                 .tresAcquisitionDateT(retrievedObject.getTresCreationDate())
-                .tresBillYear(retrievedObject.getTresBillYear())
-                .tresBillCode(retrievedObject.getTresBillCode())
+                .tresBillYear(treasuryOrigin != TreasuryOrigin.TREASURY_XLS ? retrievedObject.getTresBillYear() : null)
+                .tresBillCode(treasuryOrigin != TreasuryOrigin.TREASURY_XLS ? retrievedObject.getTresBillCode() : null)
```

Comment:
```suggestion
                .tresBillYear(!TreasuryOrigin.TREASURY_XLS.equals(treasuryOrigin) ? retrievedObject.getTresBillYear() : null)
                .tresBillCode(!TreasuryOrigin.TREASURY_XLS.equals(treasuryOrigin) ? retrievedObject.getTresBillCode() : null)
```

we could also put an if before the return to check the origin just once...